content/docs/fairing/azure/_index.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Fairing on Azure"
+description = "Documentation for Kubeflow Fairing on Azure Kubernetes Service (AKS)"
+weight = 45
++++
+
+This page documents how to run the [Fairing prediction sample notebook](https://github.com/kubeflow/fairing/blob/master/examples/prediction/xgboost-high-level-apis.ipynb) on Azure in a notebook hosted in Kubeflow.
+
+While the documentation shows how to run the sample on [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-in/services/kubernetes-service/), the steps should work on any standard Kubernetes cluster.
```

Comment:
Regarding the statement "the steps should work on any standard Kubernetes cluster", is this statement accurate? The instructions appear to be specific to configuring access to Kubernetes clusters running on Azure. 

If you have recommendations for how to configure Fairing for Kubernetes in general, it would be great to get your thoughts on the [guide to configuring Fairing](https://www.kubeflow.org/docs/fairing/configure-fairing/).